pkg/apiserver/registry/networkpolicy/clustergroupmember/rest.go

```diff
@@ -69,6 +70,10 @@ func (r *REST) NamespaceScoped() bool {
 	return false
 }
 
+func (r *REST) GetSingularName() string {
+	return "clustergroupmember"
```

Comment:
The object is `ClusterGroupMembers`. Like Endpoints, the singular name should also be `clustergroupmembers`.

Reply:
Fixed.